src/cephadm/cephadm

```diff
@@ -1704,6 +1717,27 @@ except ImportError:
                 loop.close()
 
 
+def _log_from_call(verbosity: 'CallVerbosity', message: str, error: bool = False) -> None:
+    if error:
+        if (
+            verbosity == CallVerbosity.VERBOSE
+            or verbosity == CallVerbosity.QUIET_UNLESS_ERROR
+            or verbosity == CallVerbosity.VERBOSE_ON_FAILURE
+        ):
+            logger.info(message)
+        elif verbosity == CallVerbosity.DEBUG:
+            logger.debug(message)
+        elif verbosity == CallVerbosity.QUIET:
+            logger.log(QUIET_LOG_LEVEL, message)
+    else:
+        if verbosity == CallVerbosity.VERBOSE:
+            logger.info(message)
+        elif verbosity == CallVerbosity.DEBUG or verbosity == CallVerbosity.VERBOSE_ON_FAILURE:
+            logger.debug(message)
```

Comment:
I don't understand why in case of `CallVerbosity.VERBOSE_ON_FAILURE:` we use debug even if no error. Shouldn't we use info in this case?

Reply:
I was mostly just maintaining the old behavior for this log setting with this one, but I think the idea is that, since by default only info level and above are logged to the console, we only log errors on that level and then non-errors go to debug level where they won't pop up in the console. Hence, it is only "verbose" (logging to the console) when there's an error.

Reply:
Ok, got it :+1: